File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        LinkedHashSet<Integer> s = new LinkedHashSet<>();
        for(int i:nums){
            s.add(i);
        }

        int index = 0;
        for(int i:s){
            nums[index] = i;
            index++;
        }

        //int k = nums.length;
        return index;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        int i=0;
        for(int j=i+1; j<nums.Length; j++){
            if(nums[i]!=nums[j]){
                nums[i+1] = nums[j];
                i++;
            }


        } 
        return (i+1);
    }
}